guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -144,16 +202,29 @@ endif::[]
 ----
 # leapp preupgrade
 ----
-The first run is expected to fail but report issues and inhibit the upgrade.
+ifdef::satellite[]
 +
+If you run {Project} in a disconnected environment, add the `--no-rhsm` and `--enablerepo` parameters:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# leapp preupgrade --no-rhsm --enablerepo BaseOS --enablerepo AppStream --enablerepo {RepoRHEL8ServerSatelliteMaintenanceProductVersion} --enablerepo {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
```

Comment:
```suggestion
# leapp preupgrade \
--no-rhsm \
--enablerepo BaseOS \
--enablerepo AppStream \
--enablerepo {RepoRHEL8ServerSatelliteMaintenanceProductVersion} \
--enablerepo {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
```

Please also sort alphabetically.

Reply:
Is it me or is `RepoRHEL8ServerSatelliteMaintenanceProductVersion` listed twice?

Reply:
@ekohl Took out the duplicate.